test/core/templaters/jinja_test.py

```diff
@@ -756,7 +756,6 @@ def test__templater_jinja_slice_template(test, result):
                 ("block_start", slice(0, 25, None), slice(0, 0, None)),
                 ("literal", slice(25, 30, None), slice(0, 5, None)),
                 ("block_start", slice(30, 47, None), slice(5, 5, None)),
-                ("literal", slice(47, 67, None), slice(5, 5, None)),
```

Comment:
I was confused at first why this changed, but we were actually testing for a bug before! If you look at the input Jinja template (lines 748-752 above), it only emits `SELECT 'E'` the _second_ time through the loop, but the old JinjaTracer thought it was emitted **both** times. (See the other occurrence of `slice(47, 67, None)` at line 764 below. That's the correct one.)